src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    value;
```

Comment:
Enum values should be in upper case as they are constants. The 'value' should be renamed to follow this convention. For example, it could be named as 'VALUE' or any other meaningful name in full uppercase letters.